libraries/AC_Fence/AC_Fence_config.h

```diff
@@ -11,5 +13,5 @@
 #endif
 
 #ifndef AC_POLYFENCE_FENCE_POINT_PROTOCOL_SUPPORT
-#define AC_POLYFENCE_FENCE_POINT_PROTOCOL_SUPPORT AP_FENCE_ENABLED
+#define AC_POLYFENCE_FENCE_POINT_PROTOCOL_SUPPORT HAL_GCS_ENABLED && AP_FENCE_ENABLED
```

Comment:
Won't the GCS_SEND_TEXT macros take care of the case when HAL_GCS_ENABLED is zero?

Is this change really needed?

Reply:
There are other bits in there that aren't gcs().send_text.

Also, the fencepoint protocol *is* mavlink, so it makes sense to gate it.

Reply:
I need coffee, sorry. I read Fence instead of Fencepoint. :) :)